src/Dapper.Oracle/Dapper.Oracle.StrongName.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="..\common.targets" />
   <PropertyGroup>
     <IsTestProject>false</IsTestProject>
-    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
```

Comment:
Please add net461 as explicit target here instead of net452. 
See https://github.com/StackExchange/Dapper/issues/1332 for details, but the gist of it is that consumers using full .net framework will end up having to add a ton of netstandard-dependencies in order for this to work.

Reply:
Added net461 to the target frameworks and committed.